api/resources_portal/views/user_organizations.py

```diff
@@ -0,0 +1,41 @@
+from rest_framework import viewsets
+from rest_framework.exceptions import MethodNotAllowed, PermissionDenied, ValidationError
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from rest_framework_extensions.mixins import NestedViewSetMixin
+
+from resources_portal.models import Organization, OrganizationUserAssociation, User
+from resources_portal.views.relation_serializers import OrganizationRelationSerializer
+
+
+class UserOrganizationViewSet(NestedViewSetMixin, viewsets.ModelViewSet):
+    queryset = Organization.objects.all().order_by("-created_at")
+    permission_classes = [IsAuthenticated]
```

Comment:
I think this is close, but I think it's an issue that any user can delete my organizations. I think we also need to check that the authenticated user is the user who's organizations are being modified. 

While thinking about this I checked out OrganizationMemberViewSet. It looks like it has a similar problem. It uses `BelongsToOrganization`, which seems appropriate for GET, but I think we need to add a `IsUserOrOwner` for the DELETE operation on that route. Would you mind including that in this PR as well?

Reply:
Whoops, I see how that's implemented in both places already. Disregard that!